Makefile

```diff
@@ -50,6 +50,9 @@ endif
 	python setup.py install
 
 q2dtor:
+ifneq ($(wildcard external/Q2DTor/.),)
+	@ echo "Q2DTor found in external/Q2DTor. Please make sure that the branch 'arkanepy3' is checked out for this."
```

Comment:
Thinking about it more, this doesn't make a lot of sense. If a user already has q2dtor, they won't have any reason to run `make q2dtor` again.

I think we can either
a) do nothing, assume that most users have not installed it yet, and tell people how to fix it if they ask
b) suggest double-checking the branch if running q2dtor fails by catching the error

Reply:
I guess I was thinking that the user might be tempted to run make q2dtor to fix issues related to q2dtor in the same way that `make clean; make` fixes a lot of issues. I was going to add code here that checked out the branch automatically, but I guess I forgot to include it.

I'll go with suggestion a) then